### PR TITLE
Updated cevelop to v1.7.1

### PR DESCRIPTION
**Pull request by tonisuter**
This pull request updates the cask for the Cevelop C++ IDE to v1.7.1.

Editing an existing cask

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by tonisuter** [comment]
Can I somehow run the checks again? It looks like the download of the app was incomplete on Travis CI (I am not sure why).

File: Casks/cevelop.rb [hunk 1]
```diff
@@ -3,8 +3,6 @@
   sha256 '5561659187360aa635ce04220544eb841e2e245f1a757c9b90f5f7a8ecef228f'
 
   url "https://www.cevelop.com/cevelop/downloads/cevelop-#{version}-macosx.cocoa.x86_64.tar.gz"
-  appcast 'https://www.cevelop.com/download/',
```
Comment (vitorgalvao) [inline Casks/cevelop.rb]: Why are you removing this? It works.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/cevelop.rb (hunk 1 above)
Comment (tonisuter) [inline Casks/cevelop.rb]: Sorry, I thought it might be the source of the error. I added it back.

**Comment by tonisuter** [comment]
Any ideas why this is happening? I downloaded the file now multiple times on different computers and in different networks. It always worked.

**Comment by vitorgalvao** [comment]
Verified locally.

**Comment by tonisuter** [comment]
Thanks!

**Pull request merged**